tailwind/styles.css

```diff
@@ -182,18 +182,8 @@ table.users td {
   border: solid 1px gray;
 }
 
-/* RESTORE OLD COLORS */
```

Comment:
Hi @rix0rrr ! Could we leave the old colors? They changed with a Tailwind update and @Felienne wanted them to stay the same.

Reply:
Interesting. We probably can, but this is not the right way to do it. Adding these here just breaks tailwind because there are now conflicts between built-in classes and new classes.

The right way to do it would be either:

* Introduce a `package.json` to fix to a specific version of Tailwind with the right colors.
* Change the Tailwind config file to restore exactly the colors we want.

I can take this as an action, should be a separate PR though.

Do you have a reference or link to the upstream Tailwind change?

Reply:
Hi @rix0rrr ! Good to know, thanks for catching this. I like the second option, so that we can keep on updating Tailwind. Unfortunately I don't know which version has the old colors, but perhaps we don't need to know it if we go with the second solution?

I can take up this task as I'm working on the PR, if you prefer.

Thanks!

Reply:
You can take it, but I do strongly feel it should be a separate PR to put proper versioning on Tailwind though, so I don't think it should go into this one.

The only reason I found out is because my Tailwind broke when I tried to rebuild the styles.

The reason we probably need the upstream change is because I believe the way to configure it in Tailwind is to give one base color, and the `-100`, `-200`, etc shades are (I think??) automatically generated from that one.

So we do need the original hex code to put back into our config.

Reply:
> automatically generated from that one.

Ah actually not true:

> Bad news, color is complicated and despite trying dozens of different tools, we've yet to find one that does a good job generating these sorts of color palettes. We picked all of Tailwind's default colors by hand, meticulously balancing them by eye and testing them in real designs to make sure we were happy with them.

https://tailwindcss.com/docs/customizing-colors